sdcm/cluster_k8s/__init__.py

```diff
@@ -2347,7 +2347,11 @@ def add_nodes(self,
 
         # Wait while whole cluster (on all racks) including new nodes are up and running
         current_dc_nodes = [node for node in self.nodes if node.dc_idx == dc_idx]
-        self.wait_for_pods_running(pods_to_wait=count, total_pods=len(current_dc_nodes) + count, dc_idx=dc_idx)
+        expected_dc_nodes_count = len(current_dc_nodes) + count
+        self.wait_for_pods_running(
+            pods_to_wait=count, total_pods=expected_dc_nodes_count, dc_idx=dc_idx)
+        self.wait_for_pods_readiness(
```

Comment:
doesn't that mean there are some unneed calls later one, that would be waiting for readiness ?, or since is a general wait for all of the pods (including loaders and others), there's not point of doing something different for db nodes ?

Reply:
Yes, it is general waiter. And it's specifics affects exactly Scylla, because exactly Scylla takes minutes betweeb `Running` and `Ready` pod states.